app/admin/survey_question.rb

```diff
@@ -0,0 +1,46 @@
+ActiveAdmin.register SurveyQuestion do
+  menu(false)
+
+  config.sort_order = "position_desc"
+
+  permit_params :prompt, :required, :survey,
+                options_attributes: [:id, :position, :value]
+
+  filter :survey
+
+  index do
+    id_column
+
+    column :prompt
+    column :required
+
+    actions do |resource|
+      link_to "Show Responses", show_responses_admin_survey_question_path(resource)
+    end
+  end
+
+  form do |f|
+    f.inputs do
+      f.input :prompt
+
+      f.has_many :options, label: "xyz", sortable: :position, allow_destroy: true do |o|
```

Comment:
What does `label: "xyz"` do?

Reply:
Whoops... it was supposed to remind me to circle back here and figure out how to change the label on a has_many fieldset.